pyproject.toml

```diff
@@ -33,24 +33,26 @@ fix = true
 select = [
   "E",       # pycodestyle (error)
   "F",       # pyflakes
+  "W",       # pycodestyle (warning)
   "B",       # flake8-bugbear
   "I",       # isort
   "RUF100",  # Unused noqa comments
   "PGH004",  # blanket noqa comments
   "UP",      # pyupgrade
+  "C4", "SIM201", "ISC001", "RET501",
```

Comment:
I quite like having comments for each rule here, like we have for the existing items in this list – but don't feel strongly about it :)